spec/api-web-contents-spec.js

```diff
@@ -1273,4 +1273,43 @@ describe('webContents module', () => {
       expect(data).to.be.an.instanceof(Buffer).that.is.not.empty()
     })
   })
+
+  // FIXME: disable during chromium update due to crash in content::WorkerScriptFetchInitation::CreateScriptLoaderOnIO
+  xdescribe('Shared Workers', () => {
+    let worker1
+    let worker2
+    let vectorOfWorkers
+
+    before(() => {
+      worker1 = new SharedWorker('../fixtures/api/shared-worker/shared-worker1.js')
+      worker2 = new SharedWorker('../fixtures/api/shared-worker/shared-worker2.js')
+      worker1.port.start()
+      worker2.port.start()
+    })
+
+    after(() => {
+      worker1.port.close()
+      worker2.port.close()
+    })
+
+    it('can get multiple shared workers', () => {
+      const contents = webContents.getAllWebContents()
+      vectorOfWorkers = contents[0].getAllSharedWorkers()
+
+      expect(vectorOfWorkers.length).to.equal(2)
+      expect(vectorOfWorkers[0].url).to.contain('shared-worker')
+      expect(vectorOfWorkers[1].url).to.contain('shared-worker')
+    })
+
+    it('can inspect a specific shared worker', (done) => {
+      const contents = webContents.getAllWebContents()
+      contents[0].once('devtools-opened', () => {
+        contents[0].closeDevTools()
+      })
+      contents[0].once('devtools-closed', () => {
+        done()
+      })
+      contents[0].inspectSharedWorkerById(vectorOfWorkers[0].id)
```

Comment:
This assumes that `vectorOfWorkers` has been initialized before running this test.
Please make the test self-contained.

Reply:
This test will still fail if the previous test hasn't been run. Tests should not depend on each other. Please make `vectorOfWorkers` a local variable of each test.

And please avoid using the word "vector", this concept doesn't exist in JavaScript. It's ok to call it simply `sharedWorkers`.